src/Perla.Plugins/Library.fsi

```diff
@@ -0,0 +1,158 @@
+namespace Perla.Plugins
+
+open System.Threading.Tasks
+
+/// This type handles the file extensions for bessed types
+/// like HTML, CSS, JS, any other extension is considered a
+/// "Custom" extension
+[<RequireQualifiedAccess; Struct>]
+type FileExtension =
```

Comment:
This feels a bit unnecessary.
Any extensions as a string would fit the bill here.
You can always use active patterns in your own code if you want to pattern-match this.
As a plugin writer, I wouldn't see much benefit in this structure.

Reply:
Hmm yes, you are probably right I'll consider taking this one out